extensions/observability-devservices/deployment/src/main/java/io/quarkus/observability/deployment/devui/ObservabilityDevUIProcessor.java

```diff
@@ -26,13 +26,13 @@ public class ObservabilityDevUIProcessor {
     @BuildStep(onlyIf = IsDevelopment.class)
     void createVersion(BuildProducer<CardPageBuildItem> cardPageBuildItemBuildProducer,
             BuildProducer<FooterPageBuildItem> footerProducer,
-            Optional<ObservabilityDevServicesConfigBuildItem> configProps) {
+            List<ObservabilityDevServicesConfigBuildItem> configProps) {
 
-        // LGTM
-        if (configProps.isPresent()) {
-            Map<String, String> runtimeConfig = configProps.get().getConfig();
-            String grafanaUrl = runtimeConfig.getOrDefault("grafana.endpoint", "");
+        for (ObservabilityDevServicesConfigBuildItem cp : configProps) {
```

Comment:
I don't see the grafana and collector endpoints in the logs you provide in the gist. We need the.

Reply:
Yeah, fixing that now ... or, it has been fixed. ;-)